althea_types/src/interop.rs

```diff
@@ -455,6 +455,11 @@ pub enum OperatorAction {
     /// Sets the min gas value to the provided value, primarily intended for use on xDai where
     /// the validators set a minimum gas price as a group without warning
     SetMinGas { new_min_gas: Uint256 },
+    /// Modifies the authorized keys used for access to routers
```

Comment:
this should note that keys in the add list are added to the authorized keys if they are not found there already and that the drop list keys are removed if found.